Just the obvious fix. GCC 14 complains about uint32_t and friends.

--- cdparanoia-III-10.2/utils.h.orig	2025-02-20 08:05:48.243980554 +0100
+++ cdparanoia-III-10.2/utils.h	2025-02-20 08:06:39.288980548 +0100
@@ -4,6 +4,7 @@
 #include <stdio.h>
 #include <errno.h>
 #include <string.h>
+#include <stdint.h>
 
 extern long buffering_write(int outf, char *buffer, long num);
 extern int buffering_close(int fd);
--- cdparanoia-III-10.2/interface/utils.h.orig	2025-02-20 08:10:05.327980523 +0100
+++ cdparanoia-III-10.2/interface/utils.h	2025-02-20 08:10:13.528980522 +0100
@@ -2,6 +2,7 @@
 #include <stdio.h>
 #include <errno.h>
 #include <string.h>
+#include <stdint.h>
 
 /* I wonder how many alignment issues this is gonna trip in the
    future...  it shouldn't trip any...  I guess we'll find out :) */
